src/Accounts/Accounts/ChangeLog.md

```diff
@@ -18,7 +18,6 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
-* Fixed typo in comments
```

Comment:
why this line need to be removed?

Reply:
I removed it because I found the change has no impact to customer. The message on release notes may confuse customer.